hdx_hapi/endpoints/middleware/util/util.py

```diff
@@ -49,24 +49,30 @@ async def track_api_call(request: Request, response: Response):
         '$browser_version': ua_browser_version,
         '$current_url': current_url,
     }
-    await send_mixpanel_event('api call', distinct_id, mixpanel_dict)
+    await send_mixpanel_event('hapi api call', distinct_id, mixpanel_dict)
 
 
 async def track_page_view(request: Request, response: Response):
-    current_url = str(request.url)
```

Comment:
I wonder whether getting current_url should be in a separate function. It seems wasteful to ignore 4 of 5 returned parameters